### PR TITLE
[client/rest]: add construction/derive

### PR DESCRIPTION
**Pull request by Jaguar0625**
problem: add initial rosetta route with tests
solution: add construction/derive

### CODE REVIEW

File: client/rest/test/plugins/rosetta/rosetta_spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Catapult.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import rosetta from '../../../src/plugins/rosetta/rosetta.js';
+import test from '../../routes/utils/routeTestUtils.js';
+import pluginTest from '../utils/pluginTestUtils.js';
+
+describe('rosetta plugin', () => {
+	pluginTest.assertThat.pluginDoesNotCreateDb(rosetta);
```
Comment (Wayonb) [inline client/rest/test/plugins/rosetta/rosetta_spec.js:27]: how to access the DB you will need to share? 🤔

**Review by Wayonb — APPROVED** [review]
👍🏾

File: client/rest/test/plugins/rosetta/rosetta_spec.js (hunk 1 above)
Comment (Jaguar0625) [reply client/rest/test/plugins/rosetta/rosetta_spec.js]: i pushed an update to forward db and updated tests

**Review by Wayonb — APPROVED** [review]
👍🏾

**Pull request merged**